xml/ha_storage_protection.xml

```diff
@@ -698,6 +698,66 @@ Timeout (msgwait)  : 10
    </procedure>
   </sect1>
 
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting up Diskless SBD</title>
+   <para>
+    SBD can be operated in a diskless mode. As diskless SBD is not able
+    to handle split-brain scenarios in a two-node cluster, it is required
+    for this fencing mechanism to use a minimum of three nodes.
+   </para>
+   <para>
+    When setting up SBD, you need to take several timeout values into account.
+    For details, see <xref linkend="sec.ha.storage.protect.watchdog.timings"/>.
+   </para>
+   <para>
+    The following steps are necessary to set up diskless storage-based
```

Comment:
This sentence is not needed, as you do not introduce a sequence of procedures but just one.